_config_tests.yml

```diff
@@ -0,0 +1,97 @@
+# Regression tests
```

Comment:
Yes, all the tests and their config files are to be moved to a new repo for Just the Docs tests.

BTW, I have a small tests repo for Just the Docs at https://pdmosses.github.io/just-the-docs-tests/. It uses GitHub Actions to build the site on GitHub Pages with Jekyll 4. I'd be happy to transfer it to the Just the Docs organization, but it will surely be better to create a fresh tests repo with a best-practice minimal config (and avoid inheriting my messy commit history).

Reply:
Great. Do you want to do that before or after this PR is merged? I personally do have permissions to create a new repository in the organization - would you want me to do that?

(also happy to just copy over your existing jtd-tests repo, sans commit history if you'd like)